src/export/ExportCL.cpp

```diff
@@ -343,15 +346,18 @@ class ExportCL final : public ExportPlugin
    };
 };
 
-ExportCL::ExportCL()
-:  ExportPlugin()
+ExportCL::ExportCL() = default;
+
+int ExportCL::GetFormatCount() const
 {
-   AddFormat();
-   SetFormat(wxT("CL"),0);
-   AddExtension(wxT(""),0);
-   SetMaxChannels(255,0);
-   SetCanMetaData(false,0);
-   SetDescription(XO("(external program)"),0);
+   return 1;
+}
+
+FormatInfo ExportCL::GetFormatInfo(int) const
+{
+   return {
+      wxT("CL"), XO("(external program)"), {""}, {}, 255, false
```

Comment:
The `{}` here and elsewhere initializes `mMask` but that member is nowhere used, so simplify

Reply:
... And so you did in the second commit.  Just squash it.